config/settings.yml

```diff
@@ -58,6 +58,7 @@ import:
         inst_id: met
     princeton_mods:
       directory: movie-posters/records/princeton
+      only: !ruby/regexp '/(.*)+.mods/'
```

Comment:
I think you want `/\.mods$/` (strings that end with `.mods`.  If you wanted to match the whole string `/*.\.mods$/` would be the way to go.